app/assets/javascripts/discourse/app/app.js

```diff
@@ -52,6 +52,11 @@ const Discourse = Application.extend({
   start() {
     document.querySelector("noscript")?.remove();
 
+    if (Error.stackTraceLimit) {
```

Comment:
Is this a safe global setting to change for all users?

Maybe it should be restricted to developers/admins? Or maybe it could be changed when you need the `Error` to raise?

Reply:
I think it should be safe, yeah. Ideally I'd like these console 'prefixes' to show up for all users (even anon) to make it easier for us to debug issues without needing admin access.